src/types.rs

```diff
@@ -232,8 +229,105 @@ pub mod consent {
 // pub type AeadNonce = [u8; 12];
 // pub type AeadTag = [u8; 16];
 
-// pub type ClientId = heapless::Vec<u8, heapless::consts::U32>;
-pub type ClientId = PathBuf;
+/**
+The "ClientId" struct is the closest equivalent to a PCB that Trussed
```

Comment:
What is a PCB?

Reply:
Process Control Block, OS "slang". If Trussed is an OS, the ClientContext struct would be roughly an equivalent to a PCB for clients ('clients' could then be understood as processes)